### DISCUSSION TITLE
Build instructions for `linux/arm64`

### DISCUSSION THREAD

**Discussion by CITIZENDOT** [zeux/meshoptimizer#686] [General]
Are there any different instructions to build for `linux/arm64` architecture?

https://github.com/zeux/meshoptimizer/blob/6d0d7ec01bbc67a464d55c13c08a600b3ba9bc3b/CMakeLists.txt#L124-L126

I think this cmake only targets `linux/amd64` architecture.

**Comment by zeux** [comment]
Should build out of the box I think, at least macOS/arm64 just works.
